python/res/fm/ecl/ecl_config.py

```diff
@@ -0,0 +1,149 @@
+import os
+import yaml
+import sys
+
+
+def _replace_env_string(value):
+    while True:
+        dollar_pos = value.find("$")
+        if dollar_pos < 0:
+            break
+
+        end_pos = value.find(":", dollar_pos)
```

Comment:
I don't fully understand why you are looking for the `' :' ` character in the `value`, shouldn't you look for a space (or, actually, anything that is not alphanumeric)

Reply:
Yes - I was totally focused on `:` separated path variables; I have added a fix.